api-gateway/kong/examples/README.md

```diff
@@ -0,0 +1,219 @@
+
+### User settings depends on ‘Role settings’
+
+Users in the left sidebar:
+
+- I created a new user with username `admin` that will have *role*  `admin`
+  - Enable the `Email Verified` option (for local testing only)
+  - Go to the `Credentials` tab
+    - Define a password and confirm it in `Password` and`Password Confirmation`, in the example the password `admin` is used (Only for local testing)
+    - Disable the `Temporary` option (for local testing only)
+  - Go to the `Role Mappings` tab
+    - In `Available Roles` select `admin` and `Add selected`, so that `admin` will be listed in `Associated Roles`.
+
+- I created a new user with username `user` that will have *role* `user`
+  - Enable the `Email Verified` option (for local testing only)
+  - Go to the `Credentials` tab
+    - Define a password and confirm it in `Password` and`Password Confirmation`, in the example the password `user` is used (Only for local testing)
+    - Enable the `Temporary` option (for local testing only)
+  - Go to the `Role Mappings` tab
+    - In `Available Roles` select`user` and `Add selected`, so that`user` will be listed in `Associated Roles`.
+
+### Kong client settings depends on ‘roles settings’
+
+Clients in the left sidebar:
+
+- Create a new `client` with the *Client ID* as`kong`
+  - Enable the `Enable` option if it is not enabled
+  - Set `Client Protocol` to`openid-connect`
+  - Set `Access Type` to`confidential`
+  - Enable the `Standard Flow Enabled` option if it is not enabled
+  - Disable the `Implicit Flow Enabled` option if it is enabled
+  - Enable the `Direct Access Grants Enabled` option if it is not enabled
+  - Enable the `Service Accounts Enabled` option if it is not enabled
+  - Enable the option `Authorization Enabled` if it is not enabled
+  - Set `Valid Redirect URIs` to` http://localhost:8000/* `
+
+#### Configuring authorization
+
+- Go to the `Authorization` tab
+  - Within `Authorization` select the`Authorization Scopes` tab
+    - Create the following scopes: `create`,`delete`, `update` and`view`.
+      - Note: **`create`  is for HTTP**POST**,  `delete` is for HTTP **DELETE**,    `update`is for HTTP **PUT** or **PATCH** and `view` is for HTTP **GET**.
+  - Within `Authorization` select the`Polices` tab
+    - Create a policy for role `admin`
+      - In `Create Policy` select`role`
+        - Set to name `Should be admin`
+        - Set the `admin` value in`Realm Roles` and select it as `Required`
+        - Set `Logic` to` Positive
+    - Create a policy for role `user`
+      - In `Create Policy` select`role`
+        - Set to name `Should be user
+        - Set the `user` value in`Realm Roles` and select it as `Required`
+        - Set `Logic` to`Positive`
+
+In the following steps It will configure a resource called `server-api-example-sec` in which a user with the role `admin` can create, edit and delete, while a user with the role user can only view:
```

Comment:
TODO: Same as passed to `config.resource`  when  add plugin `pepkong` to a service.